<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player</class>
 <widget class="QWidget" name="Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>智能音箱</string>
  </property>
  <widget class="QListWidget" name="music_listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>-20</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>智能音箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceid_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DevID:0000</string>
   </property>
  </widget>
  <widget class="QLabel" name="current_music_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设备未在线</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="prev_button">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>◀◀</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="play_button">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>▶</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="next_button">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>▶▶</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>361</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="volsub_button">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>vol-</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="vol_label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="voladd_button">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>vol+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>361</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="single_radioButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>单曲循环</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="order_radioButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>顺序播放</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
